freqtrade/freqai/prediction_models/CatboostClassifierMultiTarget.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import sys
+from pathlib import Path
+from typing import Any, Dict
+
+from catboost import CatBoostClassifier, Pool
+
+from freqtrade.freqai.base_models.BaseClassifierModel import BaseClassifierModel
+from freqtrade.freqai.base_models.FreqaiMultiOutputClassifier import FreqaiMultiOutputClassifier
+from freqtrade.freqai.data_kitchen import FreqaiDataKitchen
+
+
+logger = logging.getLogger(__name__)
+
+
+class CatboostClassifierMultiTarget(BaseClassifierModel):
```

Comment:
It should be very similar. I'd need to check if same modification of predict and predict_proba is needed.

Thanks for adding tests.

Reply:
I got it working with lightgbm, xgboost was complaining.

If a user comes to us asking for XGBoost multiclassifier -then I will sort it out. Otherwise - we let it be for now.